static/scss/notification.scss

```diff
@@ -47,3 +47,10 @@
     }
   }
 }
+
+#footer-notifications {
+  position: sticky;
+  top: 0;
+
+  font: bold 18px "Helvetica", sans-serif;
```

Comment:
Helvetica is already the standard font for the site. I don't see any reason why we should specify the font family again here